### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.RequestFailedException
  id: RequestFailedException
  children:
  - System.IdentityModel.RequestFailedException.#ctor
  - System.IdentityModel.RequestFailedException.#ctor(System.String)
  - System.IdentityModel.RequestFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.RequestFailedException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RequestFailedException
  nameWithType: RequestFailedException
  fullName: System.IdentityModel.RequestFailedException
  type: Class
  summary: "De uitzondering die wordt gegenereerd als de opgegeven aanvraag (eerste) is mislukt vanwege een externe reden die specifiek kan niet worden bepaald."
  syntax:
    content: 'public class RequestFailedException : System.IdentityModel.RequestException'
  inheritance:
  - System.Exception
  - System.IdentityModel.RequestException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.RequestFailedException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.RequestFailedException
  langs:
  - csharp
  name: RequestFailedException()
  nameWithType: RequestFailedException.RequestFailedException()
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException()
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.RequestFailedException&quot;> </xref> klasse."
  remarks: "Deze constructor initialiseert de <xref:SystemException.Message>eigenschap van het nieuwe exemplaar voor een systeem geleverd bericht met een beschrijving van de fout.</xref:SystemException.Message> Dit bericht houdt rekening met de huidige systeemcultuur.       De volgende tabel toont de eerste waarden voor het nieuwe exemplaar.      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException>| `null`.|   | <xref:System.Exception.Message>| De gelokaliseerde fouttekenreeks bericht. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: public RequestFailedException ();
    parameters: []
  overload: System.IdentityModel.RequestFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.RequestFailedException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.RequestFailedException
  langs:
  - csharp
  name: RequestFailedException(String)
  nameWithType: RequestFailedException.RequestFailedException(String)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.RequestFailedException&quot;> </xref> klasse met een opgegeven foutbericht."
  remarks: "Deze constructor initialiseert de <xref:System.Exception.Message>eigenschap van de nieuwe uitzondering via de `message` parameter.</xref:System.Exception.Message> De inhoud van `message` is bedoeld om te worden ge√Ønterpreteerd door mensen. De aanroeper van deze constructor is vereist om ervoor te zorgen dat deze tekenreeks gelokaliseerd voor de huidige systeemcultuur.       De volgende tabel toont de eerste waarden voor het nieuwe exemplaar.      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException>| `null`.|   | <xref:System.Exception.Message>| De fouttekenreeks bericht. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: public RequestFailedException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
  overload: System.IdentityModel.RequestFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.RequestFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.RequestFailedException
  langs:
  - csharp
  name: RequestFailedException(SerializationInfo,StreamingContext)
  nameWithType: RequestFailedException.RequestFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.RequestFailedException&quot;> </xref> klasse met de geserialiseerde gegevens."
  remarks: "Deze constructor is aangeroepen tijdens de deserialisatie op het object exception verzonden via een stroom opnieuw samenstellen."
  syntax:
    content: protected RequestFailedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Een <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> object dat de gegevens geserialiseerd object bevat."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Een <xref:System.Runtime.Serialization.StreamingContext>object dat de contextuele informatie over de bron- of doelserver bevat.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.RequestFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.RequestFailedException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.RequestFailedException
  langs:
  - csharp
  name: RequestFailedException(String,Exception)
  nameWithType: RequestFailedException.RequestFailedException(String,Exception)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.RequestFailedException&quot;> </xref> klasse met een opgegeven foutbericht en een verwijzing naar de interne uitzondering dat de oorzaak van deze uitzondering."
  remarks: "Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering moet een verwijzing bevatten naar de vorige uitzondering in de <xref:System.Exception.InnerException>eigenschap.</xref:System.Exception.InnerException> De <xref:System.Exception.InnerException>eigenschap dezelfde waarde die is doorgegeven aan de constructor.</xref:System.Exception.InnerException>       De volgende tabel toont de eerste waarden voor het nieuwe exemplaar.      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException>| De verwijzing naar de interne uitzondering. |   | <xref:System.Exception.Message>| De fouttekenreeks bericht. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: public RequestFailedException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
    - id: innerException
      type: System.Exception
      description: "De <xref:System.Exception>die de oorzaak van de huidige uitzondering.</xref:System.Exception> Als de `innerException` parameter is niet <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, de huidige uitzondering is opgetreden tijdens een <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blok die verantwoordelijk is voor de interne uitzondering."
  overload: System.IdentityModel.RequestFailedException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.IdentityModel.RequestException
  isExternal: false
  name: System.IdentityModel.RequestException
- uid: System.IdentityModel.RequestFailedException.#ctor
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException()
  nameWithType: RequestFailedException.RequestFailedException()
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException()
- uid: System.IdentityModel.RequestFailedException.#ctor(System.String)
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException(String)
  nameWithType: RequestFailedException.RequestFailedException(String)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.RequestFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException(SerializationInfo,StreamingContext)
  nameWithType: RequestFailedException.RequestFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.RequestFailedException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException(String,Exception)
  nameWithType: RequestFailedException.RequestFailedException(String,Exception)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.IdentityModel.RequestFailedException.#ctor*
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException
  nameWithType: RequestFailedException.RequestFailedException
